consensus/polybft/fsm.go

```diff
@@ -43,6 +43,7 @@ var (
 		"is either nil or it does not match the received one")
 	errValidatorSetDeltaMismatch        = errors.New("validator set delta mismatch")
 	errValidatorsUpdateInNonEpochEnding = errors.New("trying to update validator set in a non epoch ending block")
+	zeroNonce                           = types.Nonce{}
```

Comment:
Maybe put this into types.go and reuse it from there?

Reply:
fixed: https://github.com/0xPolygon/polygon-edge/pull/1508/commits/2dbdab21c1a4f5e8d850222fce96e40590deb492